### PR TITLE
Update fullstory-saml-tutorial.md

### PR DESCRIPTION
**Pull request by amyvofs**
We download and upload the IdP-generated metadata.xml file to Fullstory when we first set it up. Once the certificate expires later on, we just download the PEM cert file to update it.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@jeevansd 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/identity/saas-apps/fullstory-saml-tutorial.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ In this section, you'll enable B.Simon to use Microsoft Entra single sign-on by
 
 ## Configure Fullstory SAML SSO
 
-To configure single sign-on on **Fullstory SAML** side, you need to send the downloaded **Certificate (PEM)** and appropriate copied URLs from Microsoft Entra admin center to [Fullstory SAML support team](mailto:support@fullstory.com). They set this setting to have the SAML SSO connection set properly on both sides. For more information, please refer [this](https://help.fullstory.com/hc/articles/360020623014-How-do-I-configure-SSO) document.
+To configure single sign-on on **Fullstory SAML** side, you need to the downloaded the **Federated Certificate (XML)**  copy and paste the contents of your IdP-generated metadata.xml file into Fullstory to complete the configuration. For more information, please refer [this](https://help.fullstory.com/hc/articles/360020623014-How-do-I-configure-SSO) document.
```
Comment (rosianahm) [inline docs/identity/saas-apps/fullstory-saml-tutorial.md:121]: Hi @amyvofs kindly check and update the grammar mistake in this sentence.

**Review by amyvofs — COMMENTED** [review]
fixed grammar

**Comment by jeevansd** [comment]
#sign-off

**Pull request merged**